dev-support/test-patch.d/whitespace.sh

```diff
@@ -16,6 +16,24 @@
 
 add_plugin whitespace
 
+
+function whitespace_linecomment_reporter
+{
+  local file=$1
+  local comment=$@
```

Comment:
shellcheck: 17: warning: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate. [SC2124]